src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -84,21 +84,27 @@ export class SchoolSelectField extends React.Component {
 
   handleOptionClick = ({ address1, address2, address3, city, facilityCode, name, state, zip, country }) => {
     this.props.selectInstitution({ address1, address2, address3, city, facilityCode, name, state });
-    const backendFriendlyCountries = {
-      USA: 'United States'
+    const isDomesticAddress = country === 'USA';
+    let street2 = address2;
+    if (address3) {
+      street2 = `${street2} ${address3}`;
```

Comment:
We might want to have the backend handle this as they're already performing other schema related translations.

Reply:
Absolutely, if we add a `street3` to the schema.